Documentation/Compass/README.md

```diff
@@ -23,20 +23,23 @@ Compass:
     /// a direction toward true West, etc.).
     /// - Parameters:
     ///   - rotation: The rotation whose value determines the heading of the compass.
-    ///   - mapViewProxy: The proxy to provide access to map view operations.
-    public init(rotation: Double?, mapViewProxy: MapViewProxy)
+    ///   - mapViewProxy: The proxy to provide access to map view operations. If `mapViewProxy`
+    ///   is non-`nil`, the proxy will be used to set the rotation to `.zero` when the compass is
+    ///   tapped on. Otherwise, the `action` will be called (set using the `action` view modifier).
+    public init(rotation: Double?, mapViewProxy: MapViewProxy? = nil)
```

Comment:
This change makes the init dual purpose which I find confusing.

I would rather have 2 inits that had 2 different behaviors.  One you provide a proxy, the other you provide an action. 

WIth this you don't have undefined or non-working behavior available if I pass `nil` proxy but don't provide an action:

```swift
// No need to set an action - everything is done via provided proxy
public init(rotation: Double?, mapViewProxy: MapViewProxy)

// You must provide an action
public init(rotation: Double?, action: @escaping () -> Void)
```

Reply:
Yea, I like that better. Thanks.